### DISCUSSION TITLE
Ipad: Running code-server via ssh tunnel

### DISCUSSION THREAD

**Discussion by equilibri0** [coder/code-server#2450] [Show and tell]
Hey,

ssh tunnels via ipad are possible, contrary to what the guide.md says. 

Using the app Blink Shell (commercial app) ssh tunnels can be established similar to a desktop pc.
https://blink.sh/docs/advanced/advanced-ssh#tunnels

I think I read somewhere that other SSH clients for the ipad should also be able to do this.

**Comment by bpmct** [comment]
Just noticed this, awesome!!
